classes/componentsupport/mod_page.php

```diff
@@ -53,6 +72,13 @@ public static function get_img_file(array $pathcomponents) {
         return $file;
     }
 
+    /**
+     * Get's an image optimised path.
+     *
+     * @param array $pathcomponents
+     * @param int $maxwidth
+     * @return string
+     */
```

Comment:
Inherited method should not have comment.

Reply:
33 Errors  https://moodle.org/plugins/filter_imageopt/3.9.0.0/24112?smurf=html#phpdoc
vs
0 Errors https://github.com/ewallah/moodle-filter_imageopt/runs/4702891371?check_suite_focus=true#step:9:1

Reply:
I'm just going by Moodle's documented coding standards which state:

"You must include a description even if it appears to be obvious from the @param and/or @return lines.

An exception is made for **overridden methods** which make no change to the meaning of the parent method and maintain the same arguments/return values. **In this case you should omit the comment completely**"

https://docs.moodle.org/dev/Coding_style#Functions

I wonder if those errors would show if we documented the functions in an interface instead of the base class?

BTW - I do really appreciate your work on this.